mautic/mautic
# Segments mapping if contact is created or identified

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR add segments mapping for new contacts and identified contacts.

#### Steps to test this PR:
1.  Apply PR
2.  Go to Configurations > Contacts settings - see new Segments mapping area
3. Setup segments for new/identified contact
4.  Try test 
- create contact from Mautic
- create from form
- create via import
etc..
5. Check If segments were added to contact

![image](https://user-images.githubusercontent.com/462477/28943129-e86d4db2-789d-11e7-831a-d61aece88e2a.png)
